ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```
@@ -7,6 +7,11 @@
 # The script exit 0 if fail to create catalogsource. this allows  the other test can be executed continuously
 set -u
 
+if [[ "${SKIP_QE_APP_REGISTRY:-false}" == "true" ]]; then
+  echo "====> SKIP_QE_APP_REGISTRY=true, skipping deploy qe catalogsource"
```

Shellcheck (0.11.0)

[error] This is a unicode space. Delete and retype it.

(SC1018)